tests/test_fields.py

```diff
@@ -21,12 +23,28 @@
 )
 
 
+if PY3:
+    unicode = str
+
+
+class TestUtilsMixin(object):
+    def assertCountEqual(self, a, b):
+        """Assert a and b are equal in length."""
+        return len(a) == len(b)
+
+
+@contextmanager
+def sub_test(field):
+    """Implement sub_test."""
+    yield
```

Comment:
I'd be interested to see how it behaves where it fails. On python 3 I'd expect to see a traceback for each iteration.